packages/clerk-js/src/core/services/authentication/AuthenticationService.ts

```diff
@@ -35,6 +35,11 @@ export class AuthenticationService {
     this.setClientUatCookieForDevelopmentInstances();
   }
 
+  public setAuthCookiesFromToken(token: string): void {
```

Comment:
```suggestion
  public onTokenUpdate(token: string): void {
```

Reply:
Why name this Service method based on where it's gonna be used and not based on what it does?